config/manager/kustomization.yaml

```diff
@@ -19,8 +19,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/kiegroup/kogito-serverless-operator-nightly
-  newTag: 1.35.0-snapshot
+  newName: 192.168.49.2:5000/kogito-serverless-operator
```

Comment:
@radtriste is this change expected?

Reply:
@spolti indeed that should not. gonna review that

Reply:
https://issues.redhat.com/browse/KOGITO-8821